packages/modular-scripts/src/build/index.ts

```diff
@@ -129,6 +130,27 @@ async function buildApp(target: string) {
     }
   }
 
+  // Add dependencies from source and bundled dependencies to target package.json
+  const packageDependencies = await getPackageDependencies(target);
+  const targetPackageJson = (await fs.readJSON(
+    path.join(targetDirectory, 'package.json'),
+  )) as CoreProperties;
+  targetPackageJson.dependencies = packageDependencies;
+  targetPackageJson.bundledDependencies = Object.keys(packageDependencies);
+  // Copy selected fields of package.json over
+  await fs.writeJSON(
+    path.join(paths.appBuild, 'package.json'),
+    {
+      name: targetPackageJson.name,
+      version: targetPackageJson.version,
+      license: targetPackageJson.license,
+      modular: targetPackageJson.modular,
+      module: targetPackageJson.module,
```

Comment:
This will be undefined, but probably best to remove?

Reply:
Done